server/text_generation_server/models/__init__.py

```diff
@@ -366,12 +368,33 @@ def get_model(
     max_input_tokens: int,
 ) -> Model:
     global FLASH_ATTENTION
+    global USE_CUSTOM_MODELING
 
     config_dict, _ = PretrainedConfig.get_config_dict(
         model_id, revision=revision, trust_remote_code=trust_remote_code
     )
     model_type = config_dict.get("model_type", None)
 
+    transformers_causal_lm_class = CausalLM
+    if not USE_CUSTOM_MODELING:
+        logger.info(
```

Comment:
No need to have 3 messages, that's spammy.

0 is quite fine by me (other log messages will rise in various implementations making it clear what's happening.)